provision/kubernetes/cluster.go

```diff
@@ -135,11 +137,12 @@ func NewClusterClient(clust *provTypes.Cluster) (*ClusterClient, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &ClusterClient{
+	clusterClient := &ClusterClient{
```

Comment:
What's the point of this change?

Reply:
Oops, no point, it was from an earlier attempt.